src/vue/Components/Options/Settings.vue

```diff
@@ -48,27 +59,61 @@
     import SettingsService from '@js/Services/SettingsService';
     import ToastService from '@js/Services/ToastService';
     import SliderField from "@vue/Components/Form/SliderField";
+    import SelectField from "@vue/Components/Form/SelectField";
     import HelpText from "@vue/Components/Options/Setting/HelpText";
 
     export default {
-        components: {HelpText, SliderField, Translate},
+        components: {HelpText, SliderField, SelectField, Translate},
         data() {
             return {
-                autoclose     : false,
-                autosubmit    : false,
-                autofill      : false,
-                basicAuth     : false,
-                compromised   : false,
-                notifyPwNew   : false,
-                relatedSearch : false,
-                notifyPwUpdate: false
+                autoclose        : false,
+                autosubmit       : false,
+                autofill         : false,
+                basicAuth        : false,
+                compromised      : false,
+                notifyPwNew      : false,
+                relatedSearch    : false,
+                notifyPwUpdate   : false,
+                recSearchOption  : 'host',
+                recSearchRows    : 8
             };
         },
 
         created() {
             this.loadData();
         },
 
+        computed: {
+            recommendationOptions() {
+                return [
+                    {
+                        id   : 'domain',
+                        label: 'LabelSearchRecommendationDomain'
+                    },
+                    {
+                        id   : 'host',
+                        label: 'LabelSearchRecommendationHost'
+                    },
+                    {
+                        id   : 'hostport',
+                        label: 'LabelSearchRecommendationHostPort'
+                    },
+                    {
+                        id   : 'exact',
+                        label: 'LabelSearchRecommendationExact'
+                    }
+                ];
+            },
+            recommendationMaxRows() {
+                var i = 1;
+                var result = [];
+                for(i =1; i <= 20; i++) {
+                    result.push({id: i, label: "LabelSearchRecommendationRows" + i.toString()})
```

Comment:
You can pass an array as label and then use a variable in the language tag.

See https://github.com/marius-wieschollek/passwords-webextension/blob/master/src/vue/Components/Accounts/Account.vue#L65 for a similar use case.